src/models/resources/actors_decode.js

```diff
@@ -0,0 +1,18 @@
+const heroTokensBad = {
+  "*Hero009*"   : "Krul",
+  "*Hero010*"   : "Skaarf",
+  "*Sayoc*"     : "Taka",
+  "*Hero016*"   : "Rona",
+};
+
+export function decodeActorName(token) {
+  // Returns the hero name, given a token.
+  //
+  // For "*Hero009*", etc., returns "Krul", using heroTokensBad look-up table.
+
+  let match = /^\*(.*)\*$/.exec(token);
+  if (match) return (heroTokensBad[token] || match[1]);
```

Comment:
Given the current code style, can you wrap this with braces?

Reply:
Can you give me an example where this code style is in use?